User/src/main/java/com/example/user/service/UserService.java

```diff
@@ -0,0 +1,70 @@
+package com.example.user.service;
+
+import com.example.user.Repository.UserRepository;
+import com.example.user.exceptions.ErrorSubmissionException;
+import com.example.user.model.User;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor
+public class UserService {
+    private final UserRepository userRepository;
+    public User getUserById(long id){
+        User user=userRepository.existsById(id) ? userRepository.findById(id) : null;
+        if(user!=null){
+            return user;
+        }
+        else{
+            throw new ErrorSubmissionException("Not found user id");
+        }
+    }
+
+    public void registerUser(User userData){
+        if(userData.getFirstName().isEmpty() || userData.getLastName().isEmpty() || userData.getEmail().isEmpty() || userData.getPassw().isEmpty()){
+            throw new ErrorSubmissionException("Empty fields");
+        }
+        else if(userRepository.existsByEmail(userData.getEmail()) || userRepository.existsByPassw((userData.getPassw()))){
+            throw new ErrorSubmissionException("Email or Password already exists");
+        }
+        else {
+            User user=new User();
```

Comment:
Po zrobieniu tej klasy REgister do rejestracji git by było przerzucić to do nowej klasy RegisterToUserMapper, która przyjmowałaby Register a oddawała User. Poza tym dodaj konstruktor żebyś nie musiał ustawiać wszystkich wartości osobno